modules/post/windows/manage/mssql_local_auth_bypass.rb

```diff
@@ -43,8 +43,8 @@ def initialize(info = {})
 
     register_options(
       [
-        OptString.new('DB_USERNAME', [true, 'New sysadmin login', '']),
-        OptString.new('DB_PASSWORD', [true, 'Password for new sysadmin login', '']),
+        OptString.new('DB_USERNAME', [true, 'New sysadmin login', nil]),
+        OptString.new('DB_PASSWORD', [true, 'Password for new sysadmin login', nil]),
```

Comment:
If you ran this module, the module would run with an empty db_username and db_password - which wouldn't work